e2e/nomostest/new.go

```diff
@@ -369,11 +366,19 @@ func FreshTestEnv(t nomostesting.NTB, opts *ntopts.New) *NT {
 		}
 	})
 
-	if err := installConfigSync(nt, opts.Nomos); err != nil {
-		nt.T.Fatal(err)
+	tg := taskgroup.New()
```

Comment:
Might be able to install the git server in parallel as well

Reply:
/hold

I think this might be worth adding

Reply:
turns out it's a PITA. I need to rewrite all the git setup func to return error instead of fatal.

Reply:
That's fine, this is a good improvement so we can add git server as a followup

/unhold

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1020